crates/test/mock-tendermint-proxy/src/proxy.rs

```diff
@@ -52,8 +52,8 @@ impl TestNodeProxy {
         Box::new(move |block| inner.on_block(block))
     }
 
-    /// Returns the latest block height.
-    fn latest_block_height(&self) -> tendermint::block::Height {
+    /// Returns the last committed block height.
+    fn last_block_height(&self) -> tendermint::block::Height {
```

Comment:
Crucially the [tendermint/cometbft module docs](https://docs.rs/tendermint/0.34.0/tendermint/abci/response/struct.Info.html#structfield.last_block_height) define `last_block_height` as "The latest block for which the app has called `Commit`" which tracks with how it's being used here.